<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>一键烧录工具</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/brush.png</normaloff>:/brush.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,1,0">
     <item>
      <widget class="QComboBox" name="firm_box">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="com_box">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gobtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="erasebtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>擦除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textOut">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progbar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>gobtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>start_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>erasebtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>start_erase_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start_btn_clicked()</slot>
  <slot>start_erase_clicked()</slot>
  <slot>refresh_port_list()</slot>
 </slots>
</ui>
